other/field-types/joinByArray.md

```diff
@@ -4,19 +4,9 @@ A `joinByArray` field expresses a one-to-many relationship between this type of
 
 For instance, if `product` pieces have a `joinByArray` field called `_fabrics` that relates them to `fabric` pieces, then the related `fabric` objects will be available as the `._fabrics` array property of each product.
```

Comment:
`._fabrics` may be clearer as "the `product._fabrics` property"

Reply:
updated.